src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsController.ts

```diff
@@ -199,6 +203,24 @@ export class InlineCompletionsController extends Disposable {
 		this._register(new InlineCompletionsHintsWidget(this.editor, this.model, this.instantiationService));
 	}
 
+	private provideScreenReaderHint(): void {
+		const showHoverKeybinding = this._keybindingService.lookupKeybinding('editor.action.showHover');
+		const accessibleViewKeybinding = this._keybindingService.lookupKeybinding('editor.action.accessibleView');
+		if (this.configurationService.getValue('accessibility.verbosity.inlineCompletions')) {
```

Comment:
Editor contributions shouldn't go through the configuration service. Instead, they should use the editor options (`this.editor.getOptions(...)``).

Reply:
it's used like that here though 🤔 
https://github.com/microsoft/vscode/blob/28f2011cd72ad009b09aae4afe97644d270cb4d9/src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsController.ts#L153

Reply:
> it's used like that here though 🤔

This is a very hacky dev-only setting that I didn't want to pull through the editor settings.

> we want this to align with the other verbosity settings

The proper way would be to route these verbosity settings through the editor.
You can see how this is done with the diff editor verbosity setting.

Reply:
the diff editor verbosity setting also uses the configuration service 

https://github.com/microsoft/vscode/blob/8cc9d27ac8dcc6d633166943132725b622db3fda/src/vs/workbench/contrib/notebook/browser/diff/diffComponents.ts#L619

Reply:
https://github.com/microsoft/vscode/blob/8cc9d27ac8dcc6d633166943132725b622db3fda/src/vs/editor/browser/widget/diffReview.ts#L181-L185